### PR TITLE
Fix CICS ENQ Command

### PR DESCRIPTION
**Pull request by KutluOzel-b**
## Checklist:
- [x] Each of my commits contains one meaningful change
- [x] I have performed rebase of my branch on top of the development
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

**Comment by KutluOzel-b** [comment]
@chacebot @mm-broadcom @ilidio-lopes Can you please review ?

File: server/engine/src/main/java/org/eclipse/lsp/cobol/implicitDialects/cics/utility/CICSEnqOptionsCheckUtility.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+public class CICSEnqOptionsCheckUtility extends CICSOptionsCheckBaseUtility {
+
+    public static final int RULE_INDEX = RULE_cics_enq;
+
+    private static final Map<Integer, ErrorSeverity> DUPLICATE_CHECK_OPTIONS =
+            new HashMap<Integer, ErrorSeverity>() {
+                {
+                    put(CICSLexer.ENQ, ErrorSeverity.ERROR);
+                    put(CICSLexer.RESOURCE, ErrorSeverity.ERROR);
+                    put(CICSLexer.LENGTH, ErrorSeverity.WARNING);
```
Comment (ilidio-lopes) [inline server/engine/src/main/java/org/eclipse/lsp/cobol/implicitDialects/cics/utility/CICSEnqOptionsCheckUtility.java]: Error for parametrized options LENGTH and MAXLIFETIME

File: server/engine/src/main/java/org/eclipse/lsp/cobol/implicitDialects/cics/utility/CICSEnqOptionsCheckUtility.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+     */
+    public <E extends ParserRuleContext> void checkOptions(E ctx) {
+        if (ctx.getRuleIndex() == RULE_cics_enq_opts)
+            checkEnq((CICSParser.Cics_enq_optsContext) ctx);
+
+        checkDuplicates(ctx);
+    }
+    private void checkEnq(CICSParser.Cics_enq_optsContext ctx) {
+        checkHasMandatoryOptions(ctx.RESOURCE(), ctx, "RESOURCE");
+    }
```
Comment (ilidio-lopes) [inline server/engine/src/main/java/org/eclipse/lsp/cobol/implicitDialects/cics/utility/CICSEnqOptionsCheckUtility.java:70]: UOW MAXLIFETIME and TASK are mutually exclusive

**Review by ilidio-lopes — CHANGES_REQUESTED** [review]

File: server/engine/src/main/java/org/eclipse/lsp/cobol/implicitDialects/cics/utility/CICSEnqOptionsCheckUtility.java (hunk 1 above)
Comment (KutluOzel-b) [reply server/engine/src/main/java/org/eclipse/lsp/cobol/implicitDialects/cics/utility/CICSEnqOptionsCheckUtility.java]: addressed at :[b666cd3](https://github.com/eclipse-che4z/che-che4z-lsp-for-cobol/pull/2586/commits/b666cd35bbb1ced677e56c0dd50d8c4dd69a1a91)

File: server/engine/src/main/java/org/eclipse/lsp/cobol/implicitDialects/cics/utility/CICSEnqOptionsCheckUtility.java (hunk 2 above)
Comment (KutluOzel-b) [reply server/engine/src/main/java/org/eclipse/lsp/cobol/implicitDialects/cics/utility/CICSEnqOptionsCheckUtility.java:70]: addressed at:[b666cd3](https://github.com/eclipse-che4z/che-che4z-lsp-for-cobol/pull/2586/commits/b666cd35bbb1ced677e56c0dd50d8c4dd69a1a91)

**Review by ilidio-lopes — APPROVED** [review]

**Pull request merged**